localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,344 @@
+ lord_male_naga:0 "Lord"
+ lady_female_naga:0 "Lady"
+ lordship_naga:0 "Lordship"
+ prince_male_naga:0 "Prince"
+ princess_female_naga:0 "Princess"
+ principalty_naga:0 "Principalty"
+ warlord_male_naga:0 "Warlord"
+ warlady_female_naga:0 "Warlady"
+ raj_male_naga:0 "Raj"
+ queen_female_naga:0 "Queen"
+
+ ### Humans group / Dwarf group ###
+ # Feudal
+ high_king_male:0 "High King"
+ high_queen_female:0 "High Queen"
+ high_kingdom:0 "High Kingdom"
+
+ # Republic
+ magistrate_male:0 "Magistrate"
+ magistrate_female:0 "Magistrate"
+ magistracy:0 "Magistracy"
+
+ ### Forsaken ###
+ # Republic
+ magistrate_male_forsaken:0 "Magistrate"
+ magistrate_female_forsaken:0 "Magistrate"
+ magistracy_forsaken:0 "Magistracy"
  
- warchief_male:0 "Warchief"
- warchief_female:0 "Warchief"
- warchief_orc_male:0 "Warchief"
- warchief_orc_female:0 "Warchief"
- warlord_male:0 "Warlord"
- warlord_female:0 "Warlord"
- high_overlord_male:0 "High Overlord"
- high_overlord_female:0 "High Overlord"
- overlord_male:0 "Overlord"
- overlord_female:0 "Overlord"
+ ### Stromic ###
+ # Special
+ emperor_arathor_male:0 "Emperor"
+ emperor_arathor_female:0 "Empress"
+ empire_arathor:0 "Empire"
 
- king_nightborne_male:0 "King"
- queen_nightborne_female:0 "Queen"
+ # Feudal
+ emperor_male_stromic:0 "Emperor"
+ emperor_female_stromic:0 "Empress"
+ empire_stromic:0 "Empire"
 
- grand_prince_nightborne_male:0 "Grand Prince"
- grand_princess_nightborne_female:0 "Grand Princess"
- grand_principality_nightborne:0 "Grand Principality"
+ ### Kul'tiran ###
+ # Feudal
+ lord_male_tirassian:0 "Lord"
+ lady_female_tirassian:0 "Lady"
+ admiral_male_tirassian:0 "Admiral"
+ admiral_female_tirassian:0 "Admiral"
+ admiralty_tirassian:0 "Admiralty"
+ grand_admiral_male_tirassian:0 "Grand Admiral"
+ grand_admiral_female_tirassian:0 "Grand Admiral"
+ grand_admiralty_tirassian:0 "Grand Admiralty"
 
- prince_nightborne_male:0 "Prince"
- princess_nightborne_female:0 "Princess"
- principality_nightborne:0 "Principality"
+ ### Zandalari ###
+ # Feudal
+ god_king_feudal:0 "God King"
+ god_queen_feudal:0 "God Queen"
 
- lord_nightborne_male:0 "Lord"
- lady_nightborne_female:0 "Lady"
+ prince_male_zandalari_kingdom:0 "Prince"
+ princess_female_zandalari_kingdom:0 "Princess"
+ prince_male_zandalari_empire:0 "Prince"
+ princess_female_zandalari_empire:0 "Princess"
 
+ ### Blood trolls ###
+ # Feudal
  blood_troll_male:0 "#"
- blood_troll_count_female:0 "Warmother"
- blood_troll_duke_female:0 "Grand Ma'da"
- blood_troll_queen_female:0 "Queen Ma'da"
- blood_troll_empress_female:0 "Divine Ma'da" lady_nightborne_female:0 "Lady"
+ warmother_female_blood_troll:0 "Warmother"
+ grand_mada_female_blood_troll:0 "Grand Ma'da"
+ queen_mada_female_blood_troll:0 "Queen Ma'da"
+ divine_mada_female_blood_troll:0 "Divine Ma'da" 
  
- lich_king_male:0 "Lich King"
- lich_king_female:0 "Lich Queen"
+ ### Drakkari ###
+ # Feudal
+ frost_king_feudal:0 "Frost King"
+ frost_queen_feudal:0 "Frost Queen"
+
+ ### Dragons ###
```

Comment:
It's actually better if you can make it that the highest title is Aspect and their spouses are Consorts.

Reply:
I can't make special flavorization for spouses badly. Also the name "Aspect" is expected when I finish Dragons Overhaul. Currently, I prefer to leave it like that because these are general words for dragons.
A non-aspect dragon can lead in-game dragonflight, like Nefarian. It would be weird if he was called Aspect Nefarian.